rocky/rocky/auth/remote_user.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from django.conf import settings
+from django.contrib.auth.backends import RemoteUserBackend as BaseRemoteUserBackend
+from django.contrib.auth.models import Group
+from tools.models import Organization, OrganizationMember
+
+logger = logging.getLogger(__name__)
+
+
+class RemoteUserBackend(BaseRemoteUserBackend):
+    """
+    Custom RemoteUserBackend that adds users to default organizations and groups.
+    """
+
+    def configure_user(self, request, user, created=True):
+        if not user.is_superuser and settings.REMOTE_USER_DEFAULT_ORGANIZATIONS:
+            user_orgs = [o.name for o in user.organizations]
+            for item in settings.REMOTE_USER_DEFAULT_ORGANIZATIONS:
+                organization_name, group_name = item.split(":")
+                if organization_name not in user_orgs:
+                    logger.info("Adding user '%s' to organization '%s'", user, organization_name)
+                    organization = Organization.objects.get(name=organization_name)
```

Comment:
When are the organizations created? Or do we only add organizations to this setting after they have been added in the interface?

Reply:
They need to be created by a superuser through the KAT UI before this setting can be used.